setup.py

```diff
@@ -123,6 +123,7 @@
     "sacremoses",
     "scikit-learn",
     "sentencepiece==0.1.91",
+    "soundfile",
```

Comment:
soundfile: https://github.com/bastibe/python-soundfile is needed for the slow integration tests